packages/data-explorer/src/components/dataView/TableRow.vue

```diff
@@ -1,19 +1,45 @@
 <template>
   <tr>
     <td v-if="isShop"><shopping-button :isSelected="isSelected" :id="id"></shopping-button></td>
+    <td v-if="!isShop">
```

Comment:
In the default card view i can do shopping AND crud operations. This makes the table only have one or the other. 
I argue to make in consistent. 
Especially if we (later) add options to for example mass remove items via "shopping" selections. ( then you would like to be able to add to card in tableview. )

Reply:
i'm sorry i don't understand what you are saying here, do you suggest to add crud operations to the cart view ?, i would expect there to be replaced with cart actions ( remove from cart )